jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1207,7 +1207,23 @@ YUI.add('juju-view-utils', function(Y) {
     // to be text/plain instead of it's actual type of application/yaml.
     var exportBlob = new Blob([exportData],
         {type: 'text/plain;charset=utf-8'});
-    saveAs(exportBlob, 'bundle.yaml');
+    var envName = db.environment.get('name');
+    saveAs(exportBlob, this._getExportName(envName));
+  },
+
+  /**
+    Get the export file name
+
+    @method _getExportName
+    @param {String} Enviroment name
+  */
+  utils._getExportName = function(envName) {
```

Comment:
To make this testable:

`utils._generateBundleExportFileName = function(envName, date=new Date()) {`

Then in the test you can pass in your own custom date object to get the same output.